docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1471,6 +1471,16 @@ This will use CRLs obtained from distribution points referenced in your certific
 /subsystem=elytron/trust-manager=twoWayTM:write-attribute(name=certificate-revocation-list.path, value=intermediate.crl.pem)
 ----
 
+===== Configure multiple certificate revocation lists
```

Comment:
I think it would be good to mention which attributes are supported for `certificate-revocation-list` both here and above.

It would also be good to update the documentation (or add it if it doesn't already exist) on how to configure certificate revocation lists via the Elytron authentication client.

Reply:
I have added the additional documentation.